.github/CODEOWNERS

```diff
@@ -2,5 +2,5 @@
 # Each line is a file pattern followed by one or more owners.
 
 # These owners will be the default owners for everything in the repo.
-*       @nhakmiller @lindsey-w @wey-chiang @jacknagz @kostaspap @glerb @bseb @panther-labs/detections
+*       @nhakmiller @lindsey-w @wey-chiang @jacknagz @kostaspap @bseb @panther-labs/detections
```

Comment:
Seeing an error on GitHub about these changes … is the error legit?

Reply:
It was a valid error apparently, Jack no longer has access to this repo. Removed him from Codeowners